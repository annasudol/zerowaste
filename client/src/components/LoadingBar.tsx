import * as React from "react";
import { makeStyles } from '@material-ui/core/styles';
import CircularProgress from '@material-ui/core/CircularProgress';

const useStyles = makeStyles((theme) => ({
    root: {
        display: 'flex',
        '& > * + *': {
            marginLeft: theme.spacing(2),
        },
    },
}));

export const LoadingBar: React.FunctionComponent = (): React.ReactElement => {
    const classes = useStyles();
    return (
        <div className="flex flex justify-center items-center h-screen">
            <div className={classes.root}><CircularProgress color="secondary" /></div>
        </div>
    )
}
