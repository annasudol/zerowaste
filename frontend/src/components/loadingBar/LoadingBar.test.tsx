import { shallow } from 'enzyme';
import React from 'react';
import { LoadingBar } from './LoadingBar';
import toJson from 'enzyme-to-json';

describe('component', (): void => {
  describe('LoadingBar', (): void => {
    it('match snapshot', (): void => {
      const wrapper = shallow(
        <LoadingBar />,
      );

      expect(toJson(wrapper)).toMatchSnapshot();
    });
});


});