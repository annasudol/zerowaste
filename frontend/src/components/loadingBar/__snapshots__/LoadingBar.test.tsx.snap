// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component LoadingBar match snapshot 1`] = `
<div
  className="flex flex justify-center items-center h-screen"
>
  <div
    className="makeStyles-root-1"
  >
    <WithStyles(ForwardRef(CircularProgress))
      color="secondary"
    />
  </div>
</div>
`;
